Automattic/woocommerce-payments
# Remove deprecated usages of @woocommerce/components <Card>

#### Changes proposed in this Pull Request

* Remove `@woocommerce/components` `<Card>` usages in favor of using `@wordpress/components` `<Card>`.  The next release of `@woocommerce/components` will remove `<Card>` so this change is fairly urgent.
* Add some minor styling target adjustments to ensure that the layout and design is retained.
* Remove some components that were using the old Card but also redundant and unused.
* Add a clarification the README about running the client build in watch mode.

I didn't add any specific tests to cover this, nothing made sense since its purely presentational.

The remaining question I have at this stage is, does this warrant a changelog entry? It probably wouldn't on wc-admin so I'm not sure here.


#### Testing instructions

1. A general smoke test of the app to make sure it builds and looks ok. This is just to ensure the removed components did not break anything.

2. Compare the styling and layout of the 2 cards on the `payments/connect` page. I've attached a screenshot of the **original** to show what they looked like before.

3. Test the "failed to load payments" state. I didn't adjust styling here as it looked good with the defaults. The simplest way to test this in dev (with a connected account) is to just comment out the error conditions stated [here](https://github.com/Automattic/woocommerce-payments/pull/2315/files#diff-9423cb7aeb9116bfa2708a5b69c9aeb1e0b5bf3d108c311a2841c0e708431ebaR23) and then enter the url to that page in the address bar `<hostname>/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails`

This is what the cards on the overview page look like in `develop`:

<img width="702" alt="develop-branch" src="https://user-images.githubusercontent.com/1281828/123569331-3bdcaf80-d81a-11eb-83d2-f0ed2348f743.png">



-------------------
